build

```
@@ -101,13 +102,74 @@ function needsSucrase(source, dest, path = '') {
 	return false;
 }
 
-function sucrase(src, out, opts) {
+// https://github.com/alangpierce/sucrase/blob/master/src/cli.ts
+function findFiles(options) {
+	const outDirPath = options.outDirPath;
+	const srcDirPath = options.srcDirPath;
+	const pathToSource = options.pathToSource || '..';
+
+	const extensions = options.sucraseOptions.transforms.includes('typescript') ?
+		['.ts', '.tsx'] :
+		['.js', '.jsx'];
+
+	if (!fs.existsSync(outDirPath)) {
+		fs.mkdirSync(outDirPath);
+	}
+
+	const outArr = [];
+	for (const child of fs.readdirSync(srcDirPath)) {
+		if (
+			['node_modules', '.git'].includes(child) ||
+			options.excludeDirs.includes(child)
+		) {
+			continue;
+		}
+		const srcChildPath = path.join(srcDirPath, child);
+		const outChildPath = path.join(outDirPath, child);
+		const pathToSourceChild = path.join(pathToSource, '..');
+		if (fs.statSync(srcChildPath).isDirectory()) {
+			const innerOptions = {...options};
```

```suggestion
			const innerOptions = Object.assign({}, options);
```

`Object.assign` also postdates early Node, but it's not syntactical so we'd hit the error before hitting this.

Honestly, if we want to write modern code here, we might want to put it in a different file wrapping sucrase, rather than in this file with a lot of restrictions on what syntax is allowed.

---

I took your suggestion and moved a couple functions out. Let me know if it works, or if you want me to put the lib somewhere else.